geziyor.go

```diff
@@ -349,3 +392,29 @@ func (g *Geziyor) startExporters() {
 		}()
 	}
 }
+
+func (g *Geziyor) startWorkers() {
+	if g.Opt.ConcurrentRequests != 0 {
+		for i := 0; i < g.Opt.ConcurrentRequests; i++ {
+			go func() {
```

Comment:
I'd use a waitgroup for each go function so you can wait for them all to complete.